lib/zendesk_apps_tools/command.rb

```diff
@@ -299,5 +313,25 @@ def check_for_update
         say_status 'warning', 'Unable to check for new versions of zendesk_apps_tools gem', :yellow
       end
     end
+
+    def download_scaffold(app_dir)
+      require 'open-uri'
+      require 'zip'
+      download = open("https://github.com/zendesk/app_scaffold/archive/master.zip")
+      tmpDownloadName = "scaffold-download-temp.zip"
+      IO.copy_stream(download, tmpDownloadName)
+      Zip::File.open(tmpDownloadName) do |zip_file|
+        zip_file.each do |entry|
+          filename = entry.name.sub("app_scaffold-master/","")
+          if filename != 'src/manifest.json'
+            puts "Extracting #{filename}"
+            entry.extract("#{app_dir}/#{filename}")
+          end
+        end
+      end
+      puts "Movingg manifest.json"
+      FileUtils.mv("#{app_dir}/manifest.json", "#{app_dir}/src/")
```

Comment:
This is making some assumptions about the directory structure of the app scaffold, which means we'd have to make sure they're forever in sync.

Reply:
I believe we are going with a pretty opinionated folder structure at the moment. Forever? It's a big word. But I think the doc should reflect this opinion. In the future, if the folder structure changes, so should the doc.

Reply:
I removed the hardcoded path, instead, recording the `manifest.json` file path while doing the zip extraction.
Also add an exception handler to the function
@danielbreves @vaevictis